Cargo.toml

```diff
@@ -31,6 +31,7 @@ riscv_page_tables = { path = "./riscv-page-tables" }
 riscv_pages = { path = "./riscv-pages" }
 riscv_regs = { path = "./riscv-regs" }
 s_mode_utils = { path = "./s-mode-utils" }
+umode_api = { path = "./umode-api" }
```

Comment:
Sorry, the inconsistent hyphenation of `s-mode` vs `umode` kind of bothers me :)

Reply:
:-) And rightly so. Will fix.

Reply:
Changed `umode` and `umode-api` to their hyphened counterpart.